quickstart/java/jre/virtual-assistant/README.md

```diff
@@ -56,5 +54,5 @@ See the [accompanying article](https://docs.microsoft.com/azure/cognitive-servic
 
 ## References
 
-* [Quickstart article on the SDK documentation site](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-virtual-assistant-java-jre)
-* [Speech SDK API reference for Java](https://aka.ms/csspeech/javaref)
+* [Quickstart article on the SDK documentation site](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/quickstart-voice-assistant-java-jre)
```

Comment:
>en-us [](start = 80, length = 5)

i think docs guys will flag this, as your suppose to let the browser choose the users language.